Cannot execute `make clean html` after running `make setup`

Steps to reproduce

- checkout repository
- execute `make setup`
- active venv `source .venv/bin/activate`
- execute `make clean html`
- observe error
```
Removing everything under 'build'...
Running Sphinx v4.4.0

Sphinx version error:
The sphinxcontrib.applehelp extension used by this project needs at least Sphinx v5.0; it therefore cannot be built with this version.
make: *** [html] Error 2
```
- install matching dependencies for sphinxcontib libs `pip install --force-reinstall 'sphinxcontrib-applehelp==1.0.2' 'sphinxcontrib-devhelp==1.0.2' 'sphinxcontrib-htmlhelp==2.0.0' 'sphinxcontrib-serializinghtml==1.1.5' 'sphinxcontrib-qthelp==1.0.3'`
- try again `make clean html`
```
Removing everything under 'build'...
Running Sphinx v4.4.0

Extension error:
Could not import extension sphinxcontrib.spelling (exception: The 'enchant' C library was not found and maybe needs to be installed.
See  https://pyenchant.github.io/pyenchant/install.html
for details
)
make: *** [html] Error 2
```
- install enchant `brew install enchant`
- then it works

Environment

macOS Sonoma - 14.2.1
Python 3.11

--- a/README.md
+++ b/README.md
@@ -9,6 +9,13 @@ by users like you! So, if you'd like to change or add something here, you're
 more than welcome to do so. Have a look at our [contributing guidelines][] to
 learn how. Also, have a look at the [list of guides][] people are looking for!
 
+## Prerequisites
+
+You need to have installed:
+
+-   Python (<= 3.11)
+-   [Enchant library](https://abiword.github.io/enchant/)
+
 ## Development
 
 Pushing for each and every change is fun, but can take some time. To speed up

--- a/requirements.txt
+++ b/requirements.txt
@@ -5,4 +5,9 @@ Sphinx==4.4.0
 sphinx-autobuild==2021.3.*
 sphinx-notfound-page==0.3
 sphinx-rtd-theme==1.0.*
+sphinxcontrib-applehelp==1.0.2
+sphinxcontrib-devhelp==1.0.2
+sphinxcontrib-htmlhelp==2.0.0
+sphinxcontrib-qthelp==1.0.3
+sphinxcontrib-serializinghtml==1.1.5
 sphinxcontrib-spelling==7.1.0

